modules/ossm-rn-known-issues.adoc

```diff
@@ -21,21 +21,34 @@ These limitations exist in {SMProductName}:
 
 * The first time you access related services such as {JaegerShortName} and Grafana, from the Kiali console, you must accept the certificate and re-authenticate using your {product-title} login credentials. This happens due to an issue with how the framework displays embedded pages in the console.
 
-* The Bookinfo sample application cannot be installed on IBM Z and IBM Power Systems.
+* The Bookinfo sample application cannot be installed on IBM Z and IBM Power.
 
-* WebAssembly extensions are not supported on IBM Z and IBM Power Systems.
+* WebAssembly extensions are not supported on IBM Z and IBM Power.
 
+* LuaJIT is not supported on IBM Power.
 
 [id="ossm-rn-known-issues-ossm_{context}"]
 == {SMProductShortName} known issues
 
 These are the known issues in {SMProductName}:
 
+* https://issues.redhat.com/browse/OSSM-1668[OSSM-1668]
+`jwksResolverCA` field is missing in `SMCP`.
+
+* https://issues.redhat.com/browse/OSSM-1655[OSSM-1655]
+Kiali dashboard shows error after enabling mTLS in `SMCP`.
```

Comment:
Same, we need more details here.  What configuration causes the issue?  What is the error message that is displayed so that if customers search for their error, they find this issue.

Reply:
Could we at least get the text of the error message here?  

"After enabling the spec.security.controlPlane.mtls setting in the SMCP, the Kiali console displays the following error message, "No subsets defined"."